main.go

```diff
@@ -1841,3 +1862,61 @@ func exitIfError(msg string, err error) {
 		log.WithFields(log.Fields{"err": err}).Fatalf(msg)
 	}
 }
+
+func enableDevMode(device ios.DeviceEntry, enablePostRestart bool) {
+	// Don't try to enable if it already is
+	devModeEnabled, err := imagemounter.IsDevModeEnabled(device)
+	exitIfError("Failed checking developer mode status", err)
+	if devModeEnabled {
+		log.Info("Developer mode is already enabled on the device")
+		return
+	}
+
+	// Perform the first step of developer mode enablement and wait for the device to restart
+	conn, err := amfi.New(device)
+	exitIfError("Failed connecting to amfi service", err)
+	err = conn.EnableDevMode()
+	exitIfError("Failed enabling developer mode", err)
+	log.Infof("Successfully enabled developer mode on device `%s`, device will restart", device.Properties.SerialNumber)
+
+	udid := device.Properties.SerialNumber
+	log.Info("Waiting for device to restart after enabling developer mode")
+	ticker := time.NewTicker(10 * time.Second)
+	defer ticker.Stop()
+
+	// Create a channel to signal and stop waiting for device to be available after 60 seconds
+	done := make(chan bool)
+	go func() {
+		time.Sleep(60 * time.Second)
+		done <- true
+	}()
+
+	// Loop trying to reinit the device to find out if it restarted
+WaitLoop:
+	for {
+		select {
+		case <-ticker.C:
```

Comment:
I think you could also add a <-time.After(60s) as a case here and remove the extra go routine?

Reply:
Yeah, good point, I forgot about that 😅

Reply:
Done 👌